draft-ietf-trans-rfc6962-bis.md

```diff
@@ -637,16 +637,22 @@ following requirements:
   that the Transparency Information ({{x509v3_transinfo_extension}}) extension
   MUST be omitted.
 
-* `SignedData.signerInfos` MUST contain a signature from the same (root or
-  intermediate) CA that will ultimately issue the certificate. This signature
-  indicates the CA's intent to issue the certificate. This intent is considered
-  binding (i.e., misissuance of the precertificate is considered equivalent to
-  misissuance of the certificate). (Note that, because of the structure of CMS,
-  the signature on the CMS object will not be a valid X.509v3 signature and so
-  cannot be used to construct a certificate from the precertificate).
+* `SignedData.signerInfos` MUST contain 1 `SignerInfo` with a `signature` from
```

Comment:
Nit: I notice the SignerInfo can identify the signer either by issuer and serial number (version = 1) or the subjectKeyIdentifier (version = 3).
Do we want to limit this further by requiring that the signer is identified by subjectKeyIdentifier?